ui/app/components/clients/dashboard.js

```diff
@@ -197,6 +190,7 @@ export default class Dashboard extends Component {
 
   // HELPERS
   filterByNamespace(namespace) {
```

Comment:
string interpolation also needs to happen in this same filtering function in `current.js`